lib/content-services/src/lib/dialogs/folder.dialog.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('FolderDialogComponent', () => {
         });
 
         it('should have the proper title', () => {
-            const title = fixture.debugElement.query(By.css('[mat-dialog-title]'));
+            const title = fixture.debugElement.query(By.css('[data-automation-id="adf-folder-dialog-title"]'));
```

Comment:
fixture.debugElement.query(By.css('[data-automation-id="adf-folder-dialog-title"]')); - this can be moved to function and reused as we use this selector twice